softlayer/vm/softlayer_vm_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	fakevm "github.com/maximilien/bosh-softlayer-cpi/softlayer/vm/fakes"
 	fakesutil "github.com/maximilien/bosh-softlayer-cpi/util/fakes"
 	fakeslclient "github.com/maximilien/softlayer-go/client/fakes"
+	"os"
 )
 
 var _ = Describe("SoftLayerVM", func() {
```

Comment:
What about tests for invalid mount points? Essentially I expect to see tests for the corner cases.